pydatastructs/utils/tests/test_code_quality.py

```diff
@@ -1,31 +1,36 @@
-import os, re, sys, pydatastructs, inspect
+import inspect
+import os
+import re
+
 
 def _list_files():
     root_path = os.path.abspath(
-                os.path.join(
-                os.path.split(__file__)[0],
-                os.pardir, os.pardir))
+        os.path.join(
+            os.path.split(__file__)[0],
+            os.pardir, os.pardir))
     py_files = []
     for (dirpath, _, filenames) in os.walk(root_path):
         for _file in filenames:
             if re.match(r".*\.py$", _file):
                 py_files.append(os.path.join(dirpath, _file))
     return py_files
 
+
 py_files = _list_files()
 
+
```

Comment:
Do flake8 tests fail without these?

Reply:
yes need to add space

Reply:
Yes, I guess there is a command or something that when executed applies flake8 formatting to the overall module by itself

Reply:
yes true